scripts/python/switch-cfg.py

```diff
@@ -0,0 +1,475 @@
+        except SwitchException as exc:
+            print(exc)
+
+    # Test set vlans on port channel (LAG)
+    def _set_allowed_vlans_port_channel(cfg):
+        print('\nTest set vlans on port channel')
+        cfg['operation'] = rlinput('Enter operation (ADD|ALL|EXCEPT|NONE|REMOVE): ',
+                                   cfg['operation'])
+        cfg['lag_ifc'] = rlinput('Enter port channel ifc #: ', str(cfg['lag_ifc']))
+        cfg['vlans'] = rlinput("Enter vlans (ex: '4' or '4 6' or '2-5'): ",
+                               str(cfg['vlans']))
+        try:
+            sw.allowed_vlans_port_channel(cfg['lag_ifc'], allow_op[cfg['operation']],
+                                          cfg['vlans'].split())
+            print('{} vlans {} to port channel interface {}'.format(cfg['operation'],
+                  cfg['vlans'], cfg['lag_ifc']))
+        except SwitchException as exc:
+            print(exc)
+
+    # Test remove LAG interface
+    def _delete_port_channel(cfg):
+        print('\nTest remove LAG interface')
+        cfg['lag_ifc'] = int(rlinput('Enter lag ifc #: ', str(cfg['lag_ifc'])))
+        try:
+            sw.remove_lag_interface(cfg['lag_ifc'])
+            print('Deleted lag ifc {}'.format(cfg['lag_ifc']))
+        except SwitchException as exc:
+            print(exc)
+
+    def _is_mlag(cfg):
+        print(sw.is_mlag_configured())
+
+    # Test show MLAG interfaces summary
+    def _show_mlag_ifcs(cfg):
+        print(sw.show_mlag_interfaces())
+
+    # Test create MLAG interface (MLAG port channel)
+    def _create_mlag_ifc(cfg):
+        print('\nTest create MLAG interface')
+        cfg['mlag_ifc'] = int(rlinput('Enter mlag ifc #: ', str(cfg['mlag_ifc'])))
+        try:
+            sw.create_mlag_interface(cfg['mlag_ifc'])
+            print('Created mlag ifc {}'.format(cfg['mlag_ifc']))
+        except SwitchException as exc:
+            print(exc)
+
+    def _add_ports_to_mlag_port_channel(cfg):
+        cfg['lag_ifc'] = rlinput('Enter lag ifc #: ', str(cfg['lag_ifc']))
+        cfg['ports'] = rlinput('Enter ports: ', str(cfg['ports']))
+        ports = cfg['ports'].split()
+        try:
+            sw.bind_ports_to_mlag_interface(ports, cfg['lag_ifc'])
+            print('Added ports {} to lag interface {}'.format(cfg['ports'], cfg['lag_ifc']))
+        except SwitchException as exc:
+            print(exc)
+
+    def _set_allowed_vlans_mlag_port_channel(cfg):
+        print('\nTest set vlans on port channel')
+        cfg['operation'] = rlinput('Enter operation (ADD|ALL|EXCEPT|NONE|REMOVE): ',
+                                   cfg['operation'])
+        cfg['lag_ifc'] = rlinput('Enter port channel ifc #: ', str(cfg['lag_ifc']))
+        cfg['vlans'] = rlinput("Enter vlans (ex: '4' or '4,6' or '2-5'): ",
+                               str(cfg['vlans']))
+        try:
+            sw.allowed_vlans_mlag_port_channel(cfg['lag_ifc'],
+                allow_op[cfg['operation']], cfg['vlans'].split())
```

Comment:
flake8 tox fail:
./scripts/python/switch-cfg.py:415:17: E128 continuation line under-indented for visual indent